.docker/nginx/default.conf.template

```diff
@@ -0,0 +1,17 @@
+upstream runbot {
+    server odoo:8069 weight=1 max_fails=3 fail_timeout=200m;
+}
+
+server {
+    listen 80;
+    server_name *.${NGINX_SERVER_NAME};
+
+    location / {
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header Host $host;
+        send_timeout 200m;
+        proxy_read_timeout 200m;
+        proxy_connect_timeout 200m;
+        proxy_pass http://runbot;
```

Comment:
I think we missed the 8072 longpolling part

But thinking about this... maybe we could let the nginx in the host instead since that the port 80 could be used by the deploy of the host and we could have many services there

Or maybe a redirect from the nginx host 80 to runbot nginx container

But not sure

I even shared the nginx configuration from runbot in order to have a recommended configuration file if we decide to use the host way